wagtail/images/management/commands/wagtail_update_image_renditions.py

```diff
@@ -0,0 +1,50 @@
+from django.core.management.base import BaseCommand, CommandError
+from wagtail.images.models import Rendition, Image
+
+class Command(BaseCommand):
+    help = 'Updates image renditions'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--purge', 
+            action='store_true', 
+            help='Purge and regenerate all image renditions'
+        )
+
+        parser.add_argument(
+            '--purge-only',
+            action='store_true',
+            help='Only purge all image renditions'
+        )
+
+
+    def handle(self, *args, **options):
+        renditions = Rendition.objects.all()
```

Comment:
This code will need to get the rendition model via the class method `get_rendition_model` on the image model that is resolved from `get_image_model`

This is undocumented but see the file
wagtail/images/models.py -> https://github.com/wagtail/wagtail/blob/2.16/wagtail/images/models.py#L299